lib/iris/cube.py

```diff
@@ -3421,7 +3449,7 @@ def slices_over(self, ref_to_slice):
         iris.cube.Cube.slices :
             Return an iterator of all subcubes given the coordinates or dimension indices.
 
-        """
+        """  # noqa: D214, D406, D407, D410, D411
```

Comment:
This seems to be the current standard to avoid Ruff messing with the `Attributes:` block of a `Cube` summary 👍